opencda/core/ml_libs/rl/__init__.py

```diff
@@ -0,0 +1,8 @@
+# __TITLE__ = "DI-drive"
+# __VERSION__ = "0.3.0"
+# __DESCRIPTION__ = "Decision AI Auto-Driving Platform"
+# __AUTHOR__ = "OpenDILab Contributors"
+# __AUTHOR_EMAIL__ = "opendilab.contact@gmail.com"
+# __version__ = __VERSION__
+
+SIMULATORS = ['carla']
```

Comment:
Suggest putting in the yaml file. Also since right now opencda only supports carla, thus we could also remove this hard-coded part.